nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/io/BulkConstants.java

```diff
@@ -21,31 +21,37 @@
 /**
  * Bulk constants for serialization.
  *
- * @since 10.2
+ * @since 10.3
  */
 public class BulkConstants {
 
-    // BulkStatus part
+    private BulkConstants() {
```

Comment:
Why has the private method moved to the top?

Reply:
yeah weird will fix it